### PR TITLE
File spectacular

### PR DESCRIPTION
**Pull request by koltonev**

### CODE REVIEW

File: _fs_parameters/allocation.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+method: supplier
+description: Items status within the suppliers inventory.
+right_code: |
+
+---
+
+#### Allocation
+_Supplier Fields_
+
+order_uuid
```
Comment (Gidgidonihah) [inline _fs_parameters/allocation.md]: Isn't this also required?

File: _fs_order_data/tracking.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+name:
+position: 2
+visibility: file-specs
+method:
+description: The Tracking section
+right_code: |
+
+---
+
+retailer_org_uuid
```
Comment (Gidgidonihah) [inline _fs_order_data/tracking.md]: Isn't this required?

File: _fs_order_data/invoice.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+title: Invoice
+name:
+position: 3
+visibility: file-specs
+method:
+description: The Invoice
+right_code: |
+
+---
+retailer_org_uuid
```
Comment (Gidgidonihah) [inline _fs_order_data/invoice.md]: Isn't this required?

File: _fs_order_data/invoice.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+: (string) Universal Unique Identifier for the Line Item.
+
+retailer_po
```
Comment (Gidgidonihah) [inline _fs_order_data/invoice.md]: Isn't this required?

File: _fs_order_data/invoice.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+: (string) Required The Purchase Order ID that you provide to identify your order.
+
+invoice_number
```
Comment (Gidgidonihah) [inline _fs_order_data/invoice.md]: Isn't this required?

File: _fs_parameters/allocation.md (hunk 1 above)
Comment (Gidgidonihah) [reply _fs_parameters/allocation.md]: If it's not ready for publishing, I would rather not have it in the master branch.

File: _fs_order_data/invoice.md (hunk 5 above)
Comment (koltonev) [reply _fs_order_data/invoice.md]: This was an error on my part I had all the required fields in the sample file and use case section but never added them to the data sections updating now.

**Pull request merged**